asterisk.rb

```diff
@@ -2,14 +2,14 @@
 
 class Asterisk < Formula
   homepage 'http://www.asterisk.org'
-  url 'http://downloads.asterisk.org/pub/telephony/asterisk/releases/asterisk-12.2.0.tar.gz'
-  sha1 '80c9e72f658ca171cada93770a53798bf5d7e7d7'
+  url 'http://downloads.asterisk.org/pub/telephony/asterisk/asterisk-13-current.tar.gz'
```

Comment:
You should link to a specific version of Asterisk. Otherwise as Asterisk upgrades, this will fail b/c the sha1 won't match.

Which happened recently with the release of 13.3.0 :-)